examples/tmp117_rate_and_averaging_test.py

```diff
@@ -7,38 +7,34 @@
 # such as the one built into the Mu editor.
 import time
 import board
-from adafruit_tmp117 import TMP117, AverageCount, MeasurementDelay
+import adafruit_tmp117
 
 i2c = board.I2C()  # uses board.SCL and board.SDA
 # i2c = board.STEMMA_I2C()  # For using the built-in STEMMA QT connector on a microcontroller
-tmp117 = TMP117(i2c)
+tmp117 = adafruit_tmp117.TMP117(i2c)
 
 # uncomment different options below to see how it affects the reported temperature
-# tmp117.averaged_measurements = AverageCount.AVERAGE_1X
-# tmp117.averaged_measurements = AverageCount.AVERAGE_8X
-# tmp117.averaged_measurements = AverageCount.AVERAGE_32X
-# tmp117.averaged_measurements = AverageCount.AVERAGE_64X
+# tmp117.averaged_measurements = adafruit_tmp117.AVERAGE_1X
+# tmp117.averaged_measurements = adafruit_tmp117.AVERAGE_8X
+# tmp117.averaged_measurements = adafruit_tmp117.AVERAGE_32X
+# tmp117.averaged_measurements = adafruit_tmp117.AVERAGE_64X
 
-# tmp117.measurement_delay = MeasurementDelay.DELAY_0_0015_S
-# tmp117.measurement_delay = MeasurementDelay.DELAY_0_125_S
-# tmp117.measurement_delay = MeasurementDelay.DELAY_0_250_S
-# tmp117.measurement_delay = MeasurementDelay.DELAY_0_500_S
-# tmp117.measurement_delay = MeasurementDelay.DELAY_1_S
-# tmp117.measurement_delay = MeasurementDelay.DELAY_4_S
-# tmp117.measurement_delay = MeasurementDelay.DELAY_8_S
-# tmp117.measurement_delay = MeasurementDelay.DELAY_16_S
+# tmp117.measurement_delay = adafruit_tmp117.DELAY_0_0015_S
+# tmp117.measurement_delay = adafruit_tmp117.DELAY_0_125_S
+# tmp117.measurement_delay = adafruit_tmp117.DELAY_0_250_S
+# tmp117.measurement_delay = adafruit_tmp117.DELAY_0_500_S
+# tmp117.measurement_delay = adafruit_tmp117.DELAY_1_S
+# tmp117.measurement_delay = adafruit_tmp117.DELAY_4_S
+# tmp117.measurement_delay = adafruit_tmp117.DELAY_8_S
+# tmp117.measurement_delay = adafruit_tmp117.DELAY_16_S
 
 print(
     "Number of averaged samples per measurement:",
-    AverageCount.string[tmp117.averaged_measurements],
-)
-print(
-    "Minimum time between measurements:",
-    MeasurementDelay.string[tmp117.measurement_delay],
-    "seconds",
+    tmp117.averaged_measurements,
```

Comment:
This will print the binary value now, so this print statement should be updated to reflect the new usage and print the correct number of averages.

Reply:
see https://github.com/adafruit/Adafruit_CircuitPython_TMP117/pull/15/commits/a6f19361a0b125da3e16f3e62ea43e0907b487b3